src/partition/repart.c

```diff
@@ -2273,6 +2273,11 @@ static int context_discard_partition(Context *context, Partition *p) {
                 log_info("Storage does not support discard, not discarding data in future partition %" PRIu64 ".", p->partno);
                 return 0;
         }
+        if (r == -EBUSY) {
+                /* Let's handle this gracefully: https://bugzilla.kernel.org/show_bug.cgi?id=211167 */
+                log_info("Block device busy; can't discard block device while it is mounted.");
```

Comment:
It's not clear from the message if this is a problem. Maybe say "%s: block device busy, not discarding the partition because it is mounted." ?